README.md

```diff
@@ -46,6 +46,13 @@ When reading files the API accepts several options:
   * `FAILFAST` : throws an exception when it meets corrupted records.
 * `inferSchema`: if `true`, attempts to infer an appropriate type for each resulting DataFrame column, like a boolean, numeric or date type. If `false`, all resulting columns are of string type. Default is `true`.
 * `columnNameOfCorruptRecord`: The name of new field where malformed strings are stored. Default is `_corrupt_record`.
+
+  Note: you should explicitly add `_corrupt_record` field to dataframe schema, like this:
```

Comment:
This isn't accurate for the common case though. It's added automatically when you infer the schema.
I think the error is pretty clear here, in the case you specify the schema, don't add this, and it tells you it expected it.

Reply:
Ok, it was not so obvious for me, so I added a note